### ISSUE TITLE
[Bug] What's New is visually highlighted with blue dot but Talkback does not announce "Highlighted."

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#9795]
## Summary
Visually there's a blue dot on the main menu in Fenix. It signifies that there's something new or interesting in that menu. When you swipe to the menu, Talkback reads it as "Menu. Highlighted." When you expand the menu, visually you can see the blue dot also appears overlaying the icon of the What's New menu item. But Talkback does not say "Highlighted" when you swipe to this item. So you get told that there's something highlighted in the menu but you don't get told which menu item it is.

## Steps to reproduce
Enable Talkback
Swipe to main menu and hear Talkback announces "Menu. Highlighted". 
Expand the menu and swipe to What's New and notice that Talkback does not announce "Highlighted"

### Expected behavior
Any item with the blue dot gets the announcement "Highlighted." 

### Actual behavior
An item with the blue dot does not get the announcement "Highlighted."

### Device information

* Android device: Galaxy S10e
* Fenix version: 4/5

**Comment by asadotzler** [comment]
This is an accessibility P1. The feature, guiding users to a particular item, is broken for the screen reader use case.

**Comment by mcarare** [comment]
Waiting on https://github.com/mozilla-mobile/android-components/issues/6593.

**Comment by softvision-miralobontiu** [comment]
Verified that with talkback on, the three-dot menu is read "highlighted menu", and when opening it, the "What's new" option is read "highlighted What's new".
Tested on the 4/15 Nightly build with Google Pixel (Android 10), Sony Xperia Z5 Premium (Android 7.1.1), and OnePlus 5T (Android 9).
